backend/src/main/java/com/skr/virtuallibrary/mapping/ModelMapper.java

```diff
@@ -28,12 +28,15 @@ public interface ModelMapper {
 
     BookRating toBookRatingEntity(BookRatingDto bookRatingDto);
 
-    @Mapping(target = "created", source = "auditData.createdDate")
-    @Mapping(target = "lastModified", source = "auditData.lastModifiedDate")
-    ReviewDto toReviewDto(Review review);
+    @Mapping(target = "created", source = "review.auditData.createdDate")
+    @Mapping(target = "lastModified", source = "review.auditData.lastModifiedDate")
+    @Mapping(target = "author", source = "user")
```

Comment:
Nie ma w klasie Review pola "user"

Reply:
ale w argumentach jest `User user` - to mapowanie ma już 2 argumenty